subprojects/language-scala/src/integTest/groovy/org/gradle/language/scala/CachedPlatformScalaCompileIntegrationTest.groovy

```diff
@@ -77,7 +78,7 @@ class CachedPlatformScalaCompileIntegrationTest extends AbstractCachedCompileInt
         then:
         compiledJavaClass.exists()
         compiledScalaClass.exists()
-        output.contains "Caching disabled for task ':compileMainJarMainScala': Gradle does not know how file 'build"
+        output.contains "Caching disabled for task ':compileMainJarMainScala': Gradle does not know how file '"
```

Comment:
I think we should check the file name here, not only that there are overlapping outputs.

Reply:
Re-added check for the `build` directory as it was before.